source/retryable-reads/tests/README.rst

```diff
@@ -148,14 +148,17 @@ data.
 .. _GridFSBucket spec: https://github.com/mongodb/specifications/blob/master/source/gridfs/gridfs-spec.rst#configurable-gridfsbucket-class
     
 Speeding Up Tests
------------------
+===================
 
 Drivers may benefit reducing `minHeartbeatFrequencyMS`_ in order to speed up
 tests. Python was able to decrease the run time of the tests greatly by lowering
 the SDAM's ``minHeartbeatFrequencyMS`` from 500ms to 50ms, thus decreasing the
 waiting time after a "not master" error:
 
 .. _minHeartbeatFrequencyMS: https://github.com/mongodb/specifications/blob/master/source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst#minheartbeatfrequencyms
+
+See also `Speeding Up Tests </source/transactions/tests/README.rst#speeding-up-tests>`_ in the Transactions spec tests.
+
```

Comment:
Since this document already talks about adjusting `minHeartbeatFrequencyMS`, I'd propose you modify it _and_ also suggest that drivers use a default `heartbeatFrequencyMS` value to match (which was never discussed here, but probably should have been).